scopes/extended.md

```diff
@@ -0,0 +1,94 @@
+# Extended application scope type definitions
```

Comment:
I think we should remove these entirely. They really only serve as examples and its caused quite a bit of confusion in the past.

Reply:
OK, I agree and I will remove it